pkgs/development/python-modules/torch/test-cuda.nix

```diff
@@ -0,0 +1,22 @@
+{ runCommandNoCC
+, python
+, torch
+}:
+
+runCommandNoCC "pynvml-gpu-test"
+{
+  nativeBuildInputs = [
+    (python.withPackages (_: [ torch ]))
+  ];
+  requiredSystemFeatures = [
+    "cuda"
+  ];
+} ''
+  python3 << EOF
+  import torch
+  assert torch.cuda.is_available()
+  print(f"{torch.cuda.is_available()=}")
```

Comment:
I think that's what I did in the beginning and then swapped it because when `is_available()=False` you see the message twice: once from the print, once from assert

Reply:
Ah, I see what you're saying -- that's okay then